share/models/creative.py

```diff
@@ -39,7 +39,7 @@ def normalize(self, node, graph):
     class Disambiguation:
         any = ('identifiers',)
 
-    class Meta:
+    class Meta(ShareObject.Meta):
```

Comment:
It seemed cleaner to have them inherit than set `base_manager_name` all over.

`Meta.abstract` is a [special case](https://docs.djangoproject.com/en/1.11/topics/db/models/#meta-inheritance) and is not inherited.